### PR TITLE
Allow setting of sticky headers.

### PR DESCRIPTION
**Pull request by johnou**
Motivation:

Currently the only way to set sticky headers is to use Orbit Container, it should be possible to set them when setting up Stage too.

Modification:

Add setter for execution and allow adding sticky headers to Execution.

Result:

Sticky headers can be set programmatically.

### CODE REVIEW

File: actors/stage/src/main/java/com/ea/orbit/actors/Stage.java [hunk 1]
```diff
@@ -264,7 +370,7 @@ public void setMode(final StageMode mode)
         {
             if (container != null)
             {
-                if (clusterPeer == null && !container.getClasses().stream().filter(ClusterPeer.class::isAssignableFrom).findAny().isPresent())
```
Comment (johnou) [inline actors/stage/src/main/java/com/ea/orbit/actors/Stage.java]: null check is done above.

File: actors/stage/src/main/java/com/ea/orbit/actors/Stage.java [hunk 2]
```diff
@@ -126,6 +134,104 @@
...
+        private String nodeName;
+        private StageMode mode = StageMode.HOST;
+        private int executionPoolSize = DEFAULT_EXECUTION_POOL_SIZE;
+
+        private Messaging messaging;
+
+        private List<ActorExtension> extensions = new ArrayList<>();
+        private Set<String> stickyHeaders = new HashSet<>();
+
+        public Builder clock(Clock clock) {
```
Comment (johnou) [inline actors/stage/src/main/java/com/ea/orbit/actors/Stage.java]: Unsure of what your naming strategy would be here clock vs setClock.

File: actors/stage/src/main/java/com/ea/orbit/actors/runtime/Execution.java [hunk 3]
```diff
@@ -493,6 +492,10 @@ public Activation(final ReferenceEntry entry, final Object instance)
         }
     }
 
+    public void addStickyHeaders(Collection<String> stickyHeaders)
```
Comment (johnou) [inline actors/stage/src/main/java/com/ea/orbit/actors/runtime/Execution.java]: No setter so developer doesn't accidently remove the transaction id sticky header.

**Comment by DanielSperry** [comment]
I'd like to have more builders like this :)

**Pull request merged**